src/mango/src/mango_cursor_view.erl

```diff
@@ -70,10 +70,11 @@ explain(Cursor) ->
     end.
 
 
-execute(#cursor{db = Db, index = Idx} = Cursor0, UserFun, UserAcc) ->
+execute(#cursor{db = Db, index = Idx, execution_stats = Stats} = Cursor0, UserFun, UserAcc) ->
     Cursor = Cursor0#cursor{
         user_fun = UserFun,
-        user_acc = UserAcc
+        user_acc = UserAcc,
+        execution_stats = mango_execution_stats:log_execution_start(Stats)
```

Comment:
This is minor. But I wonder if this shouldn't be done in the create function rather than execute.